src/systems/multicopter_control/LeeVelocityController.hh

```diff
@@ -49,7 +49,7 @@ namespace multicopter_control
   /// https://github.com/ethz-asl/rotors_simulator/blob/master/rotors_control/include/rotors_control/lee_position_controller.h
   /// The controller can be used to command linear velocity and yaw angle
   /// velocity expressed in the body frame.
-  class LeeVelocityController
+  class IGNITION_GAZEBO_VISIBLE LeeVelocityController
```

Comment:
This isn't installed, so I think this shouldn't be needed?

Reply:
Uh, I don't think that binary symbols work in the same way than development headers.

This file is part of the `multicopter-control` system library since MulticopterVelocityControl.hh -> #include "LeeVelocityController.hh". MulticopterVelocityControl.cc -> #include "MulticopterVelocityControl.hh" so the class symbols are going to be in the public ABI of the library. 

```
/usr/lib/x86_64-linux-gnu🔒 ❯ nm -D --demangle  libignition-gazebo3-multicopter-control-system.so | grep LeeVelocityController
00000000000796a0 T ignition::gazebo::v3::systems::multicopter_control::LeeVelocityController::MakeController(ignition::gazebo::v3::systems::multicopter_control::LeeVelocityControllerParameters const&, ignition::gazebo::v3::systems::multicopter_control::VehicleParameters const&)
00000000000781b0 T ignition::gazebo::v3::systems::multicopter_control::LeeVelocityController::InitializeParameters()
0000000000077d40 T ignition::gazebo::v3::systems::multicopter_control::LeeVelocityController::CalculateRotorVelocities(ignition::gazebo::v3::systems::multicopter_control::FrameData const&, ignition::gazebo::v3::systems::multicopter_control::EigenTwist const&, Eigen::Matrix<double, -1, 1, 0, -1, 1>&) const
0000000000077640 T ignition::gazebo::v3::systems::multicopter_control::LeeVelocityController::ComputeDesiredAngularAcc(ignition::gazebo::v3::systems::multicopter_control::FrameData const&, ignition::gazebo::v3::systems::multicopter_control::EigenTwist const&, Eigen::Matrix<double, 3, 1, 0, 3, 1> const&) const
00000000000774a0 T ignition::gazebo::v3::systems::multicopter_control::LeeVelocityController::ComputeDesiredAcceleration(ignition::gazebo::v3::systems::multicopter_control::FrameData const&, ignition::gazebo::v3::systems::multicopter_control::EigenTwist const&) const
```

Reply:
We're not expecting users to be linking against these plugin libraries, so we don't need to care about their ABI, right?

Reply:
It wouldn't be the first time I see people linking against plugins but it can be consider as an unsupported use case. I don't have too much experience with plugins to know if there are more implications but with my knowledge I agree on what you say.

Reply:
We aren't installing the headers, so I'm not sure how a downstream project would be able to use it. The intention is not to support this use case. So far, all Ignition plugins aren't meant to be extended.

Reply:
removed visibility from all the plugins in a different PR. I did the same here.